graph2tac/loader/predict_server.py

```diff
@@ -433,12 +475,19 @@ def prediction_loop(predict_server: PredictServer, context: GlobalContextMessage
 def start_prediction_loop(predict_server: PredictServer, capnp_socket: socket.socket, record_file: Optional[BinaryIO]):
     prediction_loop(predict_server, capnp_message_generator(capnp_socket, record_file))
 
+def start_prediction_loop_with_replay(predict_server: PredictServer, replay_file: Path):
+    with open(replay_file, "rb") as f:
+        prediction_loop(predict_server, capnp_message_generator_from_file(f))
+
```

Comment:
I think that `start_prediction_loop` and `start_prediction_loop_with_replay` might as well be inlined. They are not very useful.

Reply:
Eventually, I was thinking of reorganizing these functions to be methods of `PredictServer` but not necessary to bother with it for now, I think.

Reply:
If easy.  I'll inline them.  If hard, I'll let mirek do it in his refactor.